bin/src/cli.rs

```diff
@@ -160,8 +160,15 @@ pub enum SubCmd {
         )]
         json: bool,
     },
-    #[clap(name = "cluster", about = "cluster management")]
-    Cluster {
+    #[clap(name = "clusters", about = "cluster management")]
```

Comment:
Could you keep the `cluster` command instead of `clusters`. You will break scripts that use them.

Reply:
sure